docs/bulk_data.rst

```diff
@@ -0,0 +1,62 @@
+=========
+Bulk Data
+=========
+
+Mapping File
+===========
+
+Step Name: (must be unique)
+    api: [(bulk)|sobject]
```

Comment:
Add a note here that sobject isn't yet implemented but you can use this to start stubbing out your yaml files for when it is?